### PR TITLE
UIROLES-46: show ID of role in URL path when viewing role details

### PR DESCRIPTION
**Pull request by aidynoJ**
**[UIROLES-46](https://folio-org.atlassian.net/browse/UIROLES-46)** - Show ID of role in URL path when viewing role details

Screenshot 2024-05-14 at 15 55 32

### CODE REVIEW

**Review by ryandberger — APPROVED** [review]

File: src/settings/components/RoleDetails/RoleDetails.js [hunk 1]
```diff
@@ -42,7 +41,7 @@ const RoleDetails = ({ onClose, roleId }) => {
 
   const getActionMenu = () => (
     <>
-      <Button buttonStyle="dropdownItem" onClick={() => history.push(`${pathname}?layout=edit&id=${roleId}`)}>
+      <Button buttonStyle="dropdownItem" onClick={() => history.push(`/settings/authorization-roles/${roleId}?layout=edit`)}>
```
Comment (zburke) [inline src/settings/components/RoleDetails/RoleDetails.js:44]: If somebody updates `package.json` to change [stripes.route](https://github.com/folio-org/ui-authorization-roles/blob/bd1c327a24666faa0d6d6a79e0ef8362bc2a3c78/package.json#L25) to `authz-roles`, this will break. 

Please explain why this change was necessary. It feels like a step in the wrong direction.
